sources/regex.go

```diff
@@ -28,6 +28,48 @@ func init() {
 			"socks4": "/api/v1/get?type=socks4",
 			"socks5": "/api/v1/get?type=socks5",
 		}, true, 3*time.Hour),
+		regexSource(56, "https://freeproxychecker.com/result", ":", map[string]string{
+			"http":   "/socks4_proxies.txt",
+			"socks4": "/socks4_proxies.txt",
+			"socks5": "/socks5_proxies.txt",
+		}, true, 12*time.Hour),
+		regexSource(57, "https://openproxylist.xyz", ":", map[string]string{
+			"http":   "/http.txt",
+			"socks4": "/socks4.txt",
+			"socks5": "/socks5.txt",
+		}, true, 1*time.Hour),
+		regexSource(58, "https://api.proxyscrape.com/v2/?request=getproxies&protocol=", ":", map[string]string{
+			"http":   "http",
+			"socks4": "socks4",
+			"socks5": "socks5",
+		}, true, 1*time.Hour),
+		regexSource(59, "https://proxyspace.pro", ":", map[string]string{
+			"http":   "/http.txt",
+			"https": "/https.txt",
+			"socks4": "/socks4.txt",
+			"socks5": "/socks5.txt",
+		}, true, 1*time.Hour),
+		regexSource(60, "https://api.good-proxies.ru/getfree.php", ":", map[string]string{
```

Comment:
Check if it's possible to get more proxies with less naive integration and file an issue if so

Reply:
we can fetch all in 1 or fetch by type, at the moment added by type but we can fetch only twice per 5 mins